flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["i386", "aarch64"]
```

```suggestion
  "skip-arches": ["aarch64"]
```

But still it's disappointing to ignore aarch64.

---

We can try to look into it for future releases. :)